src/main/java/de/dennisguse/opentracks/data/interfaces/ExternalStorageUtil.java

```diff
@@ -0,0 +1,80 @@
+package de.dennisguse.opentracks.data.interfaces;
+
+import com.google.gson.JsonObject;
+
+import de.dennisguse.opentracks.data.models.CRUDConstants;
+
+public interface ExternalStorageUtil {
+    /**
+     * Creates a new entry in external storage.
+     *
+     * @param collection The collection in which to create the entry
+     * @param id         The ID of the entry to be updated
+     * @param jsonData   The data to be added as the new entry
+     */
+    void createEntry(final String collection, final String id, final JsonObject jsonData, final ActionCallback callback);
+
+    /**
+     * Updates an existing entry in external storage.
+     *
+     * @param collection The collection containing the entry to be updated
+     * @param id         The ID of the entry to be updated
+     * @param jsonData       The new data to be updated in the entry
```

Comment:
minor: indentation is a bit off

Reply:
nice catch! ill make a note to fix once merging:)